src/Sportalytics.API.Feed/Controllers/FeedController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Sportalytics.Application.Feed.Commands;
+using Sportalytics.Application.Feed.Queries;
+using Sportalytics.Domain.DTOs;
+
+namespace Sportalytics.API.Feed.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public sealed class FeedController(ISender sender) : ControllerBase
+{
+    [HttpPost]
```

Comment:
See REST naming conventions https://restfulapi.net/resource-naming/, check all the endpoints. Mostly you will develop RESTful APIs

Reply:
Done)